integrations-router.js

```diff
@@ -1,14 +1,52 @@
 const express = require('express');
 const {MongoClient} = require('mongodb');
+const nodemailer = require('nodemailer');
 const config = require('./config');
 
 const integrationsRouter = express.Router(); // eslint-disable-line new-cap
+const transporter = nodemailer.createTransport({
+	service: 'gmail',
+	auth: {
+		user: 'maker.lora@gmail.com',
+		pass: 'makerpanel'
```

Comment:
Put auth props into config file and get them as environment variables so they won't be hard coded.